js/web/lib/wasm/jsep/webgpu/gpu-data-manager.ts

```diff
@@ -323,13 +334,19 @@ class GpuDataManagerImpl implements GpuDataManager {
         buffer.destroy();
       });
     });
+    this.freeUniformBuffers.forEach((buffers) => {
+      buffers.forEach(buffer => {
+        buffer.destroy();
+      });
+    });
 
     this.storageCache.forEach((storage) => {
       storage.gpuData.buffer.destroy();
     });
 
     this.storageCache = new Map();
     this.freeBuffers = new Map();
+    this.freeUniformBuffers = new Map();
```

Comment:
How about 
```
    this.storageCache = null;
    this.freeBuffers = null;
    this.freeUniformBuffers = null;
```
BTW, gpuDataManager.dispose wasn't used in backedn-webgpu.

Reply:
This is good suggestion. However I tried to write it like this, and typescript reports:
> Type 'null' is not assignable to type 'Map<number, StorageCacheValue>

I don't know if there is a better way to do this. If I define `storageCache` as nullable type, every time when I use it I need to do a null check, which is not necessary. Not sure what is the best practice for this typescript scenario.

And I guess this is why assigning to new Map was there at the first place.